Src/DeUrgenta.User.Api/Extensions/AuthExtensions.cs

```diff
@@ -19,6 +19,7 @@ public static class AuthExtensions
 
         public static IServiceCollection AddBearerAuth(this IServiceCollection services, IConfiguration configuration)
         {
+            JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
```

Comment:
@idormenco we need this because the jwtbearer middleware renames the claims we construct our token with.